### ISSUE TITLE
Curve input - face output

### ISSUE THREAD

**Issue by Durman** [nortikin/sverchok#3300]
This socket does not produce any data, should be removed I suppose.

![2020-06-02_08-52-50](https://user-images.githubusercontent.com/28003269/83481344-8de3c200-a4ae-11ea-8232-5a27d3fc6e2c.png)

**Comment by zeffii** [comment]
yes probably, or renamed and hidden_safe for a while. i think it was supposed to be to pass _cross sections_ of the curve.
